Week1/homework/app.js

```diff
@@ -1,11 +1,198 @@
+
+    const bookTitle = document.createElement('h1');
+    const bookTitleText = document.createTextNode(bookObject.title);
+    bookTitle.appendChild(bookTitleText);
+
+    const bookLanguage = document.createElement('h4');
+    const bookLanguageText = document.createTextNode(bookObject.language);
+    bookLanguage.appendChild(bookLanguageText);
+
+    const bookAuthor = document.createElement('p');
+    const bookAuthorText = document.createTextNode(bookObject.author);
+    bookAuthor.appendChild(bookAuthorText);
+
+    bookDiv.appendChild(bookTitle);
+    bookDiv.appendChild(bookLanguage);
+    bookDiv.appendChild(bookAuthor);
+
+    div.appendChild(bookDiv);
+  }
+  document.body.appendChild(div);
+  /* 1.6 Beautify your html page with css (use the style.css file for that), 
+add sources and alts to each of the images. */
+
+  // check css file please
+
+  /* 1.7 Find and download book covers for each book and construct a new object 
+which has as keys the book titles again, and as value the path to the image source 
+(e.g. { harry_potter_blabla: './img/harry_potter_blabla.jpg', ... }). */
+
+  const bookCovers = {
+    harry_potter_chamber_secrets:
+      'https://vignette.wikia.nocookie.net/harrypotter/images/c/c0/ALOExwKoxdkdeBvVi7NkaFl5Wa5.jpg/revision/latest?cb=20130803163017',
+    where_the_crowdads_sing:
+      'https://www.leeskost.nl/wp-content/uploads/2019/04/WheretheCrowdadssing.jpg',
+    educated_a_memoir: 'https://images.booksense.com/images/504/590/9780399590504.jpg',
+    unfreedom_of_the_press:
+      'https://d28hgpri8am2if.cloudfront.net/book_images/onix/cvr9781476773094/unfreedom-of-the-press-9781476773094_lg.jpg',
+    howard_stern_comes_again:
+      'https://d28hgpri8am2if.cloudfront.net/book_images/onix/cvr9781501194290/howard-stern-comes-again-9781501194290_lg.jpg',
+    the_mueller_report:
+      'https://d28hgpri8am2if.cloudfront.net/book_images/onix/cvr9781982129736/the-mueller-report-9781982129736_lg.jpg',
+    daring_greatly:
+      'https://cdn.shopify.com/s/files/1/0188/4830/products/daringgreatly_1024x1024.jpg?v=1477278732',
+    liver_rescue:
+      'https://www.bigw.com.au/medias/sys_master/images/images/hb6/he6/12774022971422.jpg',
+    atomic_habits:
+      'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1535115320l/40121378._SY475_.jpg',
+    sapiens:
+      'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1433930311l/20873740._SY475_.jpg',
+  };
+
+  /* 1.8 Loop over these entries (hint: Object.keys(objectName) gives you an array containing the keys). 
+Then write a function which places an image at the corresponding li element. 
+Remember that objects are not ordered, so you cannot guarantee that the first key 
+is the first li element. (Hint: you could give each li item an id tag 
+  by modifying the function you made before.) */
+
+  const div2 = document.createElement('div');
+
+  for (const [bookKey, bookObject] of Object.entries(bookObjects)) {
```

Comment:
Here you should you use `Object.keys` to loop over the `bookCovers` object and then use the key to find the correct element and add the cover to that element.